src/udev/udev-worker.c

```diff
@@ -248,7 +248,7 @@ void udev_broadcast_result(sd_device_monitor *monitor, sd_device *dev, EventResu
                                 (void) device_add_property(dev, "UDEV_WORKER_ERRNO_NAME", str);
                         break;
                 }
-                case EVENT_RESULT_EXIT_STATUS_BASE ... EVENT_RESULT_EXIT_STATUS_MAX:
+                case (EVENT_RESULT_EXIT_STATUS_BASE+1) ... EVENT_RESULT_EXIT_STATUS_MAX:
```

Comment:
I do not like this, and I cannot catch why you want to change the code like that.

If you want to make the branch not hit with `result == 0`, then please add `assert(result != EVENT_RESULT_SUCCESS);` or `assert(result != EVENT_RESULT_EXIT_STATUS_BASE);` in the beginning of the `case`.

Reply:
Ok, done the changes